src/seamless_communication/inference/transcriber.py

```diff
@@ -280,6 +284,11 @@ def transcribe(
         src_lang: str,
         filter_width: int = 3,
         sample_rate: int = 16000,
+        denoise: bool = False,
```

Comment:
nit: I suggest to create a dataclass DenoisingConfig, where you will put all denoise_* params, and then add a single parameter denoise_config: Optionals[DenoisingConfig] here.

Reply:
fixed